spec/views/hq/users/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "hq/users/_form.html.erb", type: :view do
+
+  before :each do
+    @user = FactoryGirl.build_stubbed(:user, user_name: 'Frederick & Bloggs', email: 'fred.bloggs@example.com')
```

Comment:
Yes ok, will remove the explicity setting of user_name and email. I had set a specific value of the user name which included an ampersand to check if I needed to do anything wrt html escaping (which is done in the equivalent partners spec). But having confirmed that it's not necessary there's no longer any need to set the fields explicitly.